apps/web/src/server/applications/applications.types.d.ts

```diff
@@ -22,7 +29,7 @@ export interface Application {
 }
 
 export interface OptionsItem {
-	id: number;
+	id?: number;
```

Comment:
wouldn't you expect this to be required?

Reply:
at the moment it's not required as the options for applicant info types don't really need an id.
i'm trying to keep the OptionsItem interface the most generic as possible and then below in the file I'm being more specific with types for ZoomLevel, Region and Sector